content/installation/Teams/_index.md

```diff
@@ -50,7 +50,7 @@ We will use "App Studio" to register and install Bot to MS Teams. Please follow
 
```

Comment:
Please remove line 25 and 99 🙂 We can point to the official Ingress docs for Kubernetes instead.

Reply:
delete, the link to the k8s docs is above 👍